app/(admin)/admin/notifications/page.tsx

```diff
@@ -1,16 +1,52 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
-import { JSX } from 'react';
+'use client';
+import { JSX, useState } from 'react';
+import { sendEmailNotifications } from './actions/sendEmailNotification';
+import React from 'react';
 
 /**
  * The admin home page.
  * @returns The rendered AdminHome page.
  */
 const AdminNotifications = (): JSX.Element => {
+  const [content, setContent] = useState<string>('');
+  const participants = ['66bd072b001f6b1f6ac0'];
```

Comment:
participants is just what I named it. The createEmail function just takes in userID's and that one is mine for testing. I have another ticket where I am doing the group league of 'Email Testers'. I had to just use mine because I had to manually enter my email in mailgun, verify it, and then use it for testing this